MIDAS/src/systems.cpp

```diff
@@ -68,6 +68,7 @@ DECLARE_THREAD(gps, RocketSystems* arg) {
 
 DECLARE_THREAD(gas, RocketSystems* arg) {
     while (true) {
+        arg->rocket_state.gas.update(arg->sensors.gas.read());
         THREAD_SLEEP(16);
```

Comment:
change the thread sleep to as closely match the maximum sampling rate as possible